lib/default.nix

```diff
@@ -403,42 +403,101 @@ let
                   gawk
                   coreutils-full
                 ];
+                # This function uses $<zero-width joiner>{ instead of $${ or $\{ to avoid shell interpolation because
+                # nix will print the $$ or \{ literally, making the output more confusing.
+                # The first empty line ensures the text is not indented.
+                warnDeprecated = oldAttr: newAttr: scriptName: lib.warn ''
+
+                  The output .${oldAttr} is deprecated in favor of .${newAttr} and will be removed in
+                  version 2.0.0! To fix this issue, you need to do the following:
+                    - If you're currently calling "$‍{system.build.${oldAttr}}" as a script,
+                      use "$‍{system.build.${newAttr}}/bin/${scriptName}" instead!
+                    - If you added "system.build.${oldAttr}" to your systemPackages, users.<name>.packages
+                      or home.packages, or used it with buildEnv or mkShell, this has never worked
+                      and using "system.build.${newAttr}" will fix it.
+                '';
               in
               lib.mapAttrs throwIfNoDisksDetected {
-                destroyScript = (diskoLib.writeCheckedBash { inherit pkgs checked; }) "disko-destroy" ''
+                diskoDestroy = (diskoLib.writeCheckedBash { inherit pkgs checked; }) "/bin/disko-destroy" ''
                   export PATH=${lib.makeBinPath destroyDependencies}:$PATH
                   ${cfg.config._destroy}
                 '';
+                diskoFormat = (diskoLib.writeCheckedBash { inherit pkgs checked; }) "/bin/disko-format" ''
```

Comment:
```suggestion
                format = (diskoLib.writeCheckedBash { inherit pkgs checked; }) "/bin/disko-format" ''
```

Reply:
This is also less to type considering this is used in `nix run`.

Reply:
Also this separates the implementation from disko making it more of an interface with  potential other implementations.

Reply:
Okay, that seems reasonable to me 👍

Reply:
Okay so I updated the PR now to use the shorter names, and solved the problem in `default.nix` by using a `_cli` prefix for the new outputs that `cli.nix` uses.

I also added deprecation warnings to all of these outputs. I believe they are not used by most users, see my commit message.